src/Domain/Domain.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why was this file created in this PR?

Reply:
The .csproj file is where the configurations and references of a project are defined. In this case this is a `Library` project.